public/css/main.css

```diff
@@ -0,0 +1,194 @@
+.standalone {
+    display: block;
+}
+
+.left-panel {
+    width: 74%;
+    float: left;
```

Comment:
Let's not use floats if we can help it – try playing with CSS grid – [this](https://cssgridgarden.com/) is a really good tutorial I found